mongofiles/options_test.go

```diff
@@ -467,4 +482,29 @@ func TestPositionalArgumentParsing(t *testing.T) {
 			}
 		}
 	})
+
+	Convey("Testing 'get_regex' with '--options' should parse the regex and the options properly", t, func() {
```

Comment:
We generally don't test things like `--regexOptions` getting set to `mf.StorageOptions.RegexOptions`. We only test options parsing where we have specific code parsing the options. But `--regexOptions` is parsed entirely by the goflags library. 

I think it's fine to keep the test, but it shouldn't be in `TestPositionalArgumentParsing`. Pull it out into its own test function.

Reply:
Done.